Bixal/move.mil
# Feature 54 theme tutorial step in tutorial content type

## Checklist

I have…

      against my changes

## Summary of Changes

Styles and configures the Tutorial Content type Tutorial Step (slideshow image) to match the move.mil look and feel so that users experience a consistent style to the current site.

[Example tutorial on the LIVE site](https://www.move.mil/tutorials/returning-user-login)

## Testing

To verify the changes proposed in this pull request…

1. pull this branch to your local env
2. make cim
3. navigate to the theme directory
4. npm build
5. make cr
6. create a basic page and include the paragraph type slideshow-image
7. complete the fields with necessary content
8. on page view you should see a display that matches the screenshot below

## Screenshots

![screencapture-move-mil-localhost-8000-node-5-2018-04-05-12_12_23](https://user-images.githubusercontent.com/37077057/38378578-302f2af0-38cc-11e8-9677-6697c03de0b8.png)
